relay-dynamic-config/src/defaults.rs

```diff
@@ -622,6 +622,38 @@ fn span_metrics(
                     .always(), // already guarded by condition on metric
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/mobile.frames_delay@millisecond".into(),
```

Comment:
not sure if relevant, but as per develop spec, the SDKs now send the frame delay [in seconds](https://develop.sentry.dev/sdk/performance/span-data-conventions/#mobile) right now

Reply:
Thanks for catching that @markushi!